src/patch_parser.py

```diff
@@ -82,6 +85,15 @@ def set_previous_line(self, item):
     def get_previous_line(self):
         return self._previous_item
 
+    def get_number_of_deleted_lines(self):
+        return self._deleted_lines
+
+    def add_deleted_line(self):
+        self._deleted_lines = self._deleted_lines + 1
```

Comment:
I agree with what Daniel said above, so you might move this out or get rid of it entirely. Nevertheless, you could make this:

```
self._deleted_lines += 1
```